Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell_Tests.swift

```diff
@@ -0,0 +1,113 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import SnapshotTesting
+@testable import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerCommandCollectionViewCell_Tests: XCTestCase {
+    /// Default width for the cell.
+    private let defaultCellWidth: CGFloat = 250
+    
+    /// Default content of the view
+    private let defaultCommand = Command(name: "Default", description: "", set: "", args: "[text]")
+    
+    func test_updateContent_whenCommandIsRecognized_commandIconImageIsUsed() {
+        var config = UIConfig()
+        config.images.commandIcons = [
+            "yoda": TestImages.yoda.image
+        ]
+        config.images.messageComposerCommandFallback = TestImages.vader.image
+        
+        let view = ChatMessageComposerCommandCellView()
+        view.uiConfig = config
+        view.content = .init(name: "yoda", description: "", set: "", args: "[text]")
+        
+        view.updateContent()
+        
+        XCTAssertEqual(view.commandImageView.image, config.images.commandIcons["yoda"])
+    }
+    
+    func test_updateContent_whenCommandIsNotRecognized_fallbackCommandImageIsUsed() {
+        var config = UIConfig()
+        config.images.commandIcons = [
+            "yoda": TestImages.yoda.image
+        ]
+        config.images.messageComposerCommandFallback = TestImages.vader.image
+        
+        let view = ChatMessageComposerCommandCellView()
+        view.uiConfig = config
+        view.content = .init(name: "R2", description: "", set: "", args: "[text]")
+        
+        view.updateContent()
+        
+        XCTAssertEqual(view.commandImageView.image, config.images.messageComposerCommandFallback)
+    }
```

Comment:
nit: You can combine these tests and just re-assing the content.

Reply:
Ups, I only saw this now, merging 😅 Only saw the "ship it" in my email. I'll refactor it in my current branch 👍

Reply:
Haha, ok 👍